admin-dev/themes/new-theme/js/pages/product/edit/customizations-manager.ts

```diff
@@ -23,13 +23,24 @@
  * @license   https://opensource.org/licenses/OSL-3.0 Open Software License (OSL 3.0)
  */
 
+import EventEmitter from '@components/event-emitter';
 import ProductMap from '@pages/product/product-map';
 import ProductEventMap from '@pages/product/product-event-map';
 import ConfirmModal from '@components/modal';
 
 const {$} = window;
 
 export default class CustomizationsManager {
+  $customizationsContainer: JQuery;
+
+  $customizationFieldsList: JQuery;
+
+  eventEmitter: typeof EventEmitter;
```

Comment:
EventEmitter exposed in `@components/event-emitter` is an instance, not a class, if you used it like it somewhere, maybe it's because you imported it from the `events` package